dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/PullRequests.java

```diff
@@ -41,5 +49,34 @@ public void prepareToCreate(final Namespace ns,
         }
         ResultsProcessor.processResults(skippedRepos, exceptions, log);
     }
+
+    /**
+     * Check if the repository is onboarded to Renovate. The signal we are looking for are
+     * (1) The presence of a file where renovate configurations are stored in the repository
+     * (2) Ensuring that the file does not have the key "enabled" set to "false"
+     * @param filePaths the list that contains all the names of the files that we are searching for in the repo
+     * @param fork A GitHubContentToProcess object that contains the fork repository that is under process
+     * @return true if the file is found in the path specified and is not disabled, false otherwise
+     */
+    private boolean isRenovateEnabled(List<String> filePaths, GitHubContentToProcess fork) throws IOException {
+        for (String filePath : filePaths) {
+            try {
+                GHContent fileContent = fork.getParent().getFileContent(filePath);
+                JSONObject json;
+                try (InputStream is = fileContent.read();
+                     BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(is))) {
```

Comment:
I think the input stream is closed when the buffered reader is closed so I think you can inline `fileContent.read()` in the `InputStreamReader` constructor.